### PR TITLE
Add STL option to ignore start of programme timecode

### PR DESCRIPTION
**Pull request by TobbeEdgeware**
Solves #49 using an option parameter in STLOptions similar to what is available for Teletext.

### CODE REVIEW

File: stl.go [hunk 1]
```diff
@@ -210,6 +216,9 @@ func ReadFromSTL(i io.Reader) (o *Subtitles, err error) {
 		STLTimecodeStartOfProgramme:                         g.timecodeStartOfProgramme,
 		Title:                                               g.originalProgramTitle,
 	}
+	if opts.IgnoreTimecodeStartOfProgramme {
```
Comment (asticode) [inline stl.go:219]: I'd rather remove line 216 and instead add

```go
if !opts.IgnoreTimecodeStartOfProgramme {
  o.Metadata.STLTimecodeStartOfProgramme = g.timecodeStartOfProgramme
}
```

**Review by asticode — CHANGES_REQUESTED** [review]
Just a minor change needed

File: stl.go (hunk 1 above)
Comment (TobbeEdgeware) [reply stl.go]: OK. Makes sense. I've changed that.

**Comment by asticode** [comment]
FYI I've created a `v0.15.0` tag

**Pull request merged**